dora/tests/src/test/java/alluxio/client/fuse/dora/hdfs3/AbstractFuseHdfsIntegrationTest.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.fuse.dora.hdfs3;
+
+import static org.junit.Assert.fail;
+
+import alluxio.AlluxioTestDirectory;
+import alluxio.Constants;
+import alluxio.client.file.FileSystemContext;
+import alluxio.client.file.options.FileSystemOptions;
+import alluxio.client.file.options.UfsFileSystemOptions;
+import alluxio.client.fuse.dora.FuseUtils;
+import alluxio.conf.Configuration;
+import alluxio.conf.PropertyKey;
+import alluxio.fuse.AlluxioFuseUtils;
+import alluxio.fuse.AlluxioJniFuseFileSystem;
+import alluxio.fuse.options.FuseOptions;
+import alluxio.jnifuse.LibFuse;
+import alluxio.testutils.LocalAlluxioClusterResource;
+
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.hdfs.MiniDFSCluster;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.IOException;
+import java.util.HashSet;
+
+public class AbstractFuseHdfsIntegrationTest {
+  @Rule
+  public TemporaryFolder mTemp = new TemporaryFolder();
+  protected static final String MOUNT_POINT = AlluxioTestDirectory
+      .createTemporaryDirectory("fuse_mount").toString();
+
+  private static final String PAGING_STORE_DIR = AlluxioTestDirectory
+      .createTemporaryDirectory("ramdisk").toString();
```

Comment:
Please rename the "ramdisk" to something like "alluxio_paging_store_dir".

"ramdisk" is a confusing name, even though we are just using a local file system dir to store the cached pages.
I have been asked so many times to explain that.

Actually from Alluxio's point of view, it is just using a dir to store all the caches pages.